### DISCUSSION TITLE
Selected KEGG pathway Heatmap for each sample

### DISCUSSION THREAD

**Discussion by Deepesh123gupta** [satijalab/seurat#6517] [General]
![image](https://user-images.githubusercontent.com/62347074/192326551-84a00b4d-44f4-4d4c-a098-9a290520e0a5.png)

Fig: Heatmap showing the abundance of selected KEGG pathway in each sample.

I am trying to generate similar heatmap using mouse or human scRNA seq dataset. Someone has an idea or steps how to proceed with it ? Thanks

**Comment by yuhanH** [comment]
Hi, @Deepesh123gupta 
This is not a Seurat-related issue. I will convert this to discussion.
